src/mongoose_graphql/admin/mongoose_graphql_stanza_admin_query.erl

```diff
@@ -0,0 +1,43 @@
+-module(mongoose_graphql_stanza_admin_query).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose_logger.hrl").
+
+execute(_Ctx, _Obj, <<"getLastMessages">>, Opts) ->
```

Comment:
Please add type specs to this module

Reply:
Added. Check the commit (Looks like diff tool still shows the old diff)
https://github.com/esl/MongooseIM/pull/3483/commits/d21a51c8e89490ea832e28f45ed3e8f1dc8ab6f9